website/source/docs/cluster/bootstrapping.html.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: "docs"
+page_title: "Creating a cluster"
+sidebar_current: "docs-cluster-bootstrap"
+description: |-
+  Learn how to bootstrap a Nomad cluster.
+---
+
+# Creating a cluster
+
+Nomad clusters in production comprises of a few Nomad servers, clients and
```

Comment:
`few Nomad servers` -> `few Nomad servers (an odd number, preferrably 3 or 5, but never an even number to prevent split-brain),`